terraform/cloudfoundry/outputs.tf

```diff
@@ -11,7 +11,11 @@ output "ssh_elb_name" {
 }
 
 output "cf_root_domain" {
```

Comment:
Why we are outputting cf_root_domain here if in terraform/vpc/outputs.tf we are outputting system_dns_zone_name with the same value?